src/agenteval/summary.py

```diff
@@ -42,17 +42,21 @@ def _mean(
 
 def _safe_mean(
     xs: Sequence[float | None],
-    is_score: bool = False,
     weights: Sequence[float] | None = None,
+    replace_none: float | int | None = None,
```

Comment:
I'm picturing you are using replace_none = True and would want same for leaderboard?